samples/config/prod/m-cluster/userstore.yaml

```diff
@@ -8,14 +8,42 @@ resources:
   requests:
     memory: 8Gi
 
+# For JDK 8
 opendjJavaArgs: >
   -server -Xms8g -Xmx8g -XX:+UseCompressedOops -XX:+UseG1GC -XX:+UseNUMA 
   -XX:MaxGCPauseMillis=100 -verbose:gc -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps 
   -XX:+PrintGCDetails -XX:+PrintPromotionFailure -XX:+PrintAdaptiveSizePolicy -Xloggc:/tmp/gc.log
 
+# For JDK 11
+#opendjJavaArgs: >
+#  -server -Xms8g -Xmx8g -XX:+UseCompressedOops -XX:+UseG1GC -XX:+UseNUMA  
+#  -XX:MaxGCPauseMillis=100 -verbose:gc -Xlog:gc:/tmp/gc.log
+
 storageSize: "500Gi"
 storageClass: "fast"
 
 replicas: 2
 
-topologyKey: "failure-domain.beta.kubernetes.io/zone"
\ No newline at end of file
+topologyKey: "failure-domain.beta.kubernetes.io/zone"
+
+backup:
+  enabled: true
+  storageClass: "nfs"
+  storageSize: "20Gi"
+  schedule:  "*/30 * * * *"
+  verify: false
+  # If you want to restore from a backup pvc then provide the name below and uncomment
+  #pvcName: "bak-userstore"
+
+gcs:
+  backup: true
```

Comment:
I thought we decided not to archive the backups to GCS buckets, so gcs.backup may need to be false.